packages/core/src/components/RadioButton/__tests__/radioButton-tests.jest.tsx

```diff
@@ -1,76 +1,99 @@
-import React from "react";
+import "@testing-library/jest-dom";
 import { fireEvent, render, cleanup, screen } from "@testing-library/react";
 import RadioButton from "../RadioButton";
 
 describe("RadioButton tests", () => {
   const formName = "myForm";
-  const radiosName = "radios";
 
-  const option1Value = "1";
+  let onChangeMock1: jest.Mock;
+  let onChangeMock2: jest.Mock;
+  let onChangeMock3: jest.Mock;
+
   const option1Text = "Option 1";
-  const option2Value = "2";
   const option2Text = "Option 2";
-  const option3Value = "3";
   const option3Text = "Option 3";
 
-  let onChangeMock1: jest.Mock;
-  let onChangeMock2: jest.Mock;
-  let onChangeMock3: jest.Mock;
+  describe("With one of the radio buttons is checked by default", () => {
+    beforeEach(() => {
+      const radiosName = "radios";
 
-  beforeEach(() => {
-    onChangeMock1 = jest.fn();
-    onChangeMock2 = jest.fn();
-    onChangeMock3 = jest.fn();
+      const option1Value = "1";
+      const option2Value = "2";
+      const option3Value = "3";
```

Comment:
I agree with removing the mocks from the `beforeEach` section, I'm not sure about inserting the `*Value` inside, and also about `radiosName`, sounds like they can lay outside of this block (even in the top of the file, outside of any block)

Reply:
Done